model/host/host.go

```diff
@@ -1020,6 +1020,15 @@ func FindAllRunningParentsByContainerPool(poolId string) ([]Host, error) {
 	return Find(query)
 }
 
+// CountUninitializedParents returns the number of initializing parent host intent documents
+func CountUninitializedParents() (int, error) {
+	query := db.Query(bson.M{
+		HasContainersKey: true,
+		StatusKey:        evergreen.HostUninitialized,
+	})
+	return Count(query)
```

Comment:
I'd avoid the declaration and just do return Count(db.Query(bson.M{})), or to avoid weirdness, just

```
db.Count(Collection, bson.M{})
```